setup.py

```diff
@@ -0,0 +1,47 @@
+"""Setup for lti_consumer XBlock."""
+
+import os
+from setuptools import setup
+
+
+def package_data(pkg, roots):
+    """Generic function to find package_data.
+
+    All of the files under each of the `roots` will be declared as package
+    data for package `pkg`.
+
+    """
+    data = []
+    for root in roots:
+        for dirname, __, files in os.walk(os.path.join(pkg, root)):
+            for fname in files:
+                data.append(os.path.relpath(os.path.join(dirname, fname), pkg))
+
+    return {pkg: data}
+
+
+setup(
+    name='lti_consumer-xblock',
+    version='v1.0.0',
+    description='This XBlock implements the consumer side of the LTI specification.',
+    packages=[
+        'lti_consumer',
+    ],
+    install_requires=[
+        'lxml',
+        'bleach',
+        'oauthlib',
+        'Mako',
+        'XBlock',
+        'xblock-utils',
+    ],
```

Comment:
since the requirements.txt specifies versions and version ranges for these libraries, they should also be spelled out too in the setup.py.